tfjs-backend-webgpu/src/kernels/FromPixels.ts

```diff
@@ -46,42 +46,65 @@ export function fromPixels(args: {
   const outShape = [pixels.height, pixels.width, numChannels];
   const imageData = (pixels as ImageData | backend_util.PixelData).data;
 
-  if (env().getBool('IS_BROWSER')) {
-    if (!(pixels instanceof HTMLVideoElement) &&
-        !(pixels instanceof HTMLImageElement) &&
-        !(pixels instanceof HTMLCanvasElement) &&
-        !(pixels instanceof ImageData) && !(pixels instanceof ImageBitmap) &&
-        !(pixels.data instanceof Uint8Array)) {
-      throw new Error(
-          'pixels passed to tf.browser.fromPixels() must be either an ' +
-          `HTMLVideoElement, HTMLImageElement, HTMLCanvasElement, ImageData, ` +
-          `ImageBitmap ` +
-          `or {data: Uint32Array, width: number, height: number}, ` +
-          `but was ${(pixels as {}).constructor.name}`);
-    }
+  const isVideo = typeof (HTMLVideoElement) !== 'undefined' &&
+      pixels instanceof HTMLVideoElement;
+  const isImage = typeof (HTMLImageElement) !== 'undefined' &&
+      pixels instanceof HTMLImageElement;
+  const isCanvas = typeof (HTMLCanvasElement) !== 'undefined' &&
+      pixels instanceof HTMLCanvasElement;
+  const isImageBitmap =
+      typeof (ImageBitmap) !== 'undefined' && pixels instanceof ImageBitmap;
+  const isImageData =
+      typeof (ImageData) !== 'undefined' && pixels instanceof ImageData;
+  const isPixelData = !isImageData &&
+      typeof ((pixels as PixelData).data) !== 'undefined' &&
+      (pixels as PixelData).data instanceof Uint8Array;
+
+  if (!isVideo && !isImage && !isCanvas && !isImageData && !isImageBitmap &&
+      !isPixelData) {
+    throw new Error(
```

Comment:
nit: Maybe this checking is not needed since we have done it in the upper level ( tfjs-core https://github.com/tensorflow/tfjs/blob/38f8462fe642011ff1b7bcbb52e018f3451be58b/tfjs-core/src/ops/browser.ts#L96)